ArcGISRuntimeSDKQt_CppSamples/Features/ControlTimeExtentTimeSlider/README.md

```diff
@@ -0,0 +1,34 @@
+# Control time extent using toolkit time slider
+
+This sample demonstrates how to use the time slider from the toolkit to visualize temporal data by applying a specific time extent.
+When viewing feature layers that contain a large amount of data with timestamps, you may want to filter the data to only show data relevant to a specific time range. This can help visualize changes in the data over time and aid in making better interpretations and predictions of future trends.
+
+
+![](screenshot.png)
+
+## Use case
+
+When viewing feature layers that contain a large amount of data with timestamps, you may want to filter the data to only show data relevant to a specific time range. This can help visualize changes in the data over time and aids in making better interpretations and predictions of future trends.
```

Comment:
nitpick, but in this sample you are adding a map image layer, not a feature layer. Also, the main description seems to have duplicate information as what is in the use case section. Maybe it should only be in the use case and the first description is just one sentence?